src/news/release-6.1.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Prism Launcher Release 6.1, now available
+description: More addictive than lemonade!
+date: 2022-12-19
+release_version: "6.1"
+minimum_macos_version: 10.15.0
+mac_signature: QY1DCk7gQrngyD8qLdLY3a3GOO27NpktQo8EsJDkAV5M1RoB3krnonYL30rFJihIlNDszOvPopPRNGYJKIeADQ==
+tags:
+  - release
+---
+
+Hello!
+
+## Changelog
+
+### Changed
+
+- Move cat to the bottom right again by [@DioEgizio](https://github.com/DioEgizio) in [#612](https://github.com/PrismLauncher/PrismLauncher/pull/612)85)
+- Only show scrollbars when needed by [@leo78913](https://github.com/leo78913) in [#630](https://github.com/PrismLauncher/PrismLauncher/pull/630)
+- [Linux] Update Nix packaging by [@Scrumplex](https://github.com/Scrumplex) in [#647](https://github.com/PrismLauncher/PrismLauncher/pull/647)
+- [Windows] Improve MSVC build flags by [@getchoo](https://github.com/getchoo) in [#631](https://github.com/PrismLauncher/PrismLauncher/pull/631)
+
+### Fixed
+
+- Avoid deadlock when there are a lot of resource packs in an instance  by [@flowln](https://github.com/flowln) in [#628](https://github.com/PrismLauncher/PrismLauncher/pull/628)
+- Correctly set component versions in Modrinth pack updating by [@flowln](https://github.com/flowln) in [#636](https://github.com/PrismLauncher/PrismLauncher/pull/636)
```

Comment:
```suggestion
- Correctly set component versions in Modrinth pack updating by [@flowln](https://github.com/flowln) and [@Scrumplex](https://github.com/Scrumplex) in [#636](https://github.com/PrismLauncher/PrismLauncher/pull/636)
```

Reply:
oh GitHub API does not return co-authors. good to know